ibm/service/power/resource_ibm_pi_volume_bulk.go

```diff
@@ -0,0 +1,537 @@
+		ReadContext:   resourceIBMPIVolumeBulkRead,
+		DeleteContext: resourceIBMPIVolumeBulkDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(30 * time.Minute),
+			Delete: schema.DefaultTimeout(30 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			// Arguments
+			Arg_AffinityInstance: {
+				ConflictsWith:    []string{Arg_AffinityVolume},
+				Description:      "PVM Instance (ID or Name) to base volume affinity policy against; required if requesting 'affinity' and 'pi_affinity_volume' is not provided.",
+				DiffSuppressFunc: flex.ApplyOnce,
+				ForceNew:         true,
+				Optional:         true,
+				Type:             schema.TypeString,
+			},
+			Arg_AffinityPolicy: {
+				Description:      "Affinity policy for data volume being created; ignored if 'pi_volume_pool' provided; for policy 'affinity' requires one of 'pi_affinity_instance' or 'pi_affinity_volume' to be specified; for policy 'anti-affinity' requires one of 'pi_anti_affinity_instances' or 'pi_anti_affinity_volumes' to be specified; Allowable values: 'affinity', 'anti-affinity'.",
+				DiffSuppressFunc: flex.ApplyOnce,
+				ForceNew:         true,
+				Optional:         true,
+				Type:             schema.TypeString,
+				ValidateFunc:     validate.InvokeValidator("ibm_pi_volume", Arg_AffinityPolicy),
+			},
+			Arg_AffinityVolume: {
+				ConflictsWith:    []string{Arg_AffinityInstance},
+				Description:      "Volume (ID or Name) to base volume affinity policy against; required if requesting 'affinity' and 'pi_affinity_instance' is not provided.",
+				DiffSuppressFunc: flex.ApplyOnce,
+				ForceNew:         true,
+				Optional:         true,
+				Type:             schema.TypeString,
+			},
+			Arg_AntiAffinityInstances: {
+				ConflictsWith:    []string{Arg_AntiAffinityVolumes},
+				Description:      "List of pvmInstances to base volume anti-affinity policy against; required if requesting 'anti-affinity' and 'pi_anti_affinity_volumes' is not provided.",
+				DiffSuppressFunc: flex.ApplyOnce,
+				Elem:             &schema.Schema{Type: schema.TypeString},
+				ForceNew:         true,
+				Optional:         true,
+				Type:             schema.TypeList,
+			},
+			Arg_AntiAffinityVolumes: {
+				ConflictsWith:    []string{Arg_AntiAffinityInstances},
+				Description:      "List of volumes to base volume anti-affinity policy against; required if requesting 'anti-affinity' and 'pi_anti_affinity_instances' is not provided.",
+				DiffSuppressFunc: flex.ApplyOnce,
+				Elem:             &schema.Schema{Type: schema.TypeString},
+				ForceNew:         true,
+				Optional:         true,
+				Type:             schema.TypeList,
+			},
+			Arg_CloudInstanceID: {
+				Description:  "The GUID of the service instance associated with an account.",
+				ForceNew:     true,
+				Required:     true,
+				Type:         schema.TypeString,
+				ValidateFunc: validation.NoZeroValues,
+			},
+			Arg_Count: {
```

Comment:
I think we set a max off 500 for the count  off-prep and 250 on-prem. This could go in the SDK , but we can add the limit to doc

Reply:
added to doc